RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -2241,7 +2241,8 @@ boolean isUnclePAvailable()
 	{
 		return false;
 	}
-	return true;
+	string page_text = visit_url("place.php?whichplace=desertbeach");
+	return page_text.contains_text("You don't know where a desert beach is") == false;
```

Comment:
nit of nits: `return !page_text.contains(...)`

Reply:
```suggestion
	return !page_text.contains_text("You don't know where a desert beach is");
```